src/libraries/Common/src/Internal/Cryptography/BasicSymmetricCipher.cs

```diff
@@ -6,6 +6,7 @@
 using System.Diagnostics;
 using System.Security.Cryptography;
 
+#nullable enable
```

Comment:
I see in all files below the `#nullable enable` is before using-s. Is it style only or order important?

Reply:
Style only. But our convention is to do it before the usings.